strapi/strapi
# [TS] Move TypeScript to Peer Dependencies

### What does it do?

Remove TypeScript from the `@strapi/strapi` and `@strapi/typescript-utils` packages and add it as a peerDependency of `@strapi/typescript-utils`

### Why is it needed?

Remove coupling between Strapi and TypeScript + do not install libraries that won't be used in JavaScript Strapi applications

### How to test it?

1. Clone the repository
2. Navigate to examples/getstarted and run yarn develop => The app should start normally
3. Navigate to examples/kitchensink-ts and run yarn develop => The app should crash while starting saying that it cannot find the typescript package
4. run yarn add typescript
5. re-run yarn develop => The app should compile then start normally
